dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -364,13 +365,13 @@ public ResourceSupport findRel(HttpServletRequest request, HttpServletResponse r
      * @return
      */
     @RequestMapping(method = RequestMethod.GET, value = REGEX_REQUESTMAPPING_IDENTIFIER_AS_STRING_VERSION_STRONG +
-        "/{rel}/{relid:[\\w+\\-]+}")
+        "/{rel}/{relid}")
     public ResourceSupport findRel(HttpServletRequest request, HttpServletResponse response,
                                    @PathVariable String apiCategory,
                                    @PathVariable String model, @PathVariable String id, @PathVariable String rel,
                                    @PathVariable String relid,
                                    Pageable page, PagedResourcesAssembler assembler,
-                                   @RequestParam(required = false) String projection) {
+                                   @RequestParam(required = false) String projection) throws Throwable {
```

Comment:
I guess `throws Throwable` was left here during early developments

Reply:
This is explained at https://github.com/atmire/DSpace/blob/authority-control-bugfixes/dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java#L779